doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -441,3 +441,54 @@ underscore or by omitting the annotation to a polymorphic definition.
    semantics that the first use declares it. In this mode, the universe
    names are not associated with the definition or proof once it has been
    defined. This is meant mainly for debugging purposes.
+
+.. flag:: Private Polymorphic Universes
+
+   Consider the following definition:
```

Comment:
This option is enabled by default. I think giving a brief description would be good too. Description by examples is usually not fully satisfying.